website/i18n/en.json

```diff
@@ -321,6 +321,9 @@
       "operations/insert-segment-to-db": {
         "title": "insert-segment-to-db tool"
       },
+      "operations/kubernetes": {
```

Comment:
Hmm, I think I need to modify this backport (i just blindly did it with a script), this page isn't in the 0.16 docs...